modules/data.atmosphere/DESCRIPTION

```diff
@@ -79,4 +79,4 @@ Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
 Encoding: UTF-8
-RoxygenNote: 7.2.3
+RoxygenNote: 7.3.1
```

Comment:
Please revert Roxygen to v7.2.3. That includes rolling back your version of the roxygen2 package and rerunning make document

Reply:
Fixed